azurerm/internal/services/cosmos/tests/cosmosdb_account_resource_test.go

```diff
@@ -786,6 +786,8 @@ resource "azurerm_cosmosdb_account" "test" {
   resource_group_name = azurerm_resource_group.test.name
   offer_type          = "Standard"
   kind                = "GlobalDocumentDB"
+  
+  enable_free_tier    = false
```

Comment:
We can remove this because it defaults to `false`. Instead, we should add a test to enable/disable the free tier. If we can't disable it, we need to add `ForceNew: true` to the schema declaration

Reply:
It is possible to have only one Free Tier account and I do not think it is possible to change it. Hence, I added the ForceNew to the schema and I introduced a new test for this. I appreciate your feedback!